### PR TITLE
OSA - Remove FQDN parameters and update ensure_osa_aad flow

### PR DESCRIPTION
**Pull request by julienstroheker**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by julienstroheker** [comment]
CC @jim-minter @mjudeikis @amanohar

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -2564,8 +2568,15 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
 
     try:
         # long_running_operation_timeout=300
-        return sdk_no_wait(no_wait, client.create_or_update,
-                           resource_group_name=resource_group_name, resource_name=name, parameters=osamc)
+        result = sdk_no_wait(no_wait, client.create_or_update,
+                             resource_group_name=resource_group_name, resource_name=name, parameters=osamc)
+        result = LongRunningOperation(cmd.cli_ctx)(result)
```
Comment (mboersma) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: We don't use `result` after this point in this method. Should it be `return`ed? Why are we making a `LongRunningOperation` out of it if it isn't used?

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.3.22
+++++++
+* Removing `--fqdn` flag on az openshift commands
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: This is a breaking change and cannot be done with a simple patch version. This needs to call out the breaking change and be bumped to version 2.4.0

**Comment by julienstroheker** [comment]
@tjprescott can you give another look please ? Thanks

**Pull request merged**